### ISSUE TITLE
Accessibility: Luminosity contrast ratio of keyboard focus indicator fails to meet the required ratio of 3:1

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212317]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: VS Code 1.89
- OS Version: Windows 11 Enterprise 23H2 (OS build 22631.3296)

Copied from: https://github.com/microsoft/vscode-azureapicenter/issues/134

### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG1.4.11; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #A11yMAS; #Non Text Contrast; #A11YWCAG2.1;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Generate HTTP file" using the down arrow.
7. Verify whether the Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation" buttons fails to meet the required ratio of 3:1

### Actual Result:
Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation" buttons fails to meet the required ratio of 3:1.

### Expected Result:
Luminosity contrast ratio of keyboard focus indicator on "Generate API client", "Generate HTTP file" and "Open API documentation" buttons should be greater than or equal to required ratio of 3:1.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

### Attachment:
![image](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/ad814967-a0ce-4d5d-aea2-9b5688c4c295)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.89.0. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by formulahendry** [comment]
Tested on latest 1.89.0, still have this issue.

**Comment by AnKushSingh05** [comment]
Primary Bug:https://github.com/microsoft/vscode-azureapicenter/issues/134

**Comment by daviddossett** [comment]
Could you confirm the theme used here? I believe this was fixed for our default light/dark themes. See https://github.com/microsoft/vscode/issues/203400

**Comment by formulahendry** [comment]
> Could you confirm the theme used here? I believe this was fixed for our default light/dark themes. See #203400

@AnKushSingh05 Please confirm,

**Comment by AnKushSingh05** [comment]
> > Could you confirm the theme used here? I believe this was fixed for our default light/dark themes. See #203400
> 
> @AnKushSingh05 Please confirm,

I am using the dark visual studio theme
![image](https://github.com/microsoft/vscode/assets/124244478/acd065a7-6686-4edb-9c0c-9675a1980bde)

**Comment by meganrogge** [comment]
David is out on paternity leave.

**Comment by daviddossett** [comment]
Will be fixed via #224642
